<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowSize2D</class>
 <widget class="QDialog" name="WindowSize2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/room_size_2d.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LeftVertSize">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>90</y>
     <width>31</width>
     <height>238</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RightVertSize">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>90</y>
     <width>31</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CenterVertSize">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>25</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension_1.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TopHorSize">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>97</y>
     <width>293</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/horizontal_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CenterHorSize">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>211</y>
     <width>293</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/horizontal_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BottomHorSize">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>300</y>
     <width>293</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/horizontal_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="CenterVertSizeNum">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>186</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="CenterHorSizeNum">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="TopHorSizeNum">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="BottomHorSizeNum">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>297</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeftVertSizeNum">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="RightVertSizeNum">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_2">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
background-color: rgb(140,90,20);</string>
   </property>
   <property name="text">
    <string>НАЗАД</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_2">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
background-color: rgb(140,90,20);</string>
   </property>
   <property name="text">
    <string>РАСЧЕТ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>101</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>191</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>187</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>211</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>191</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>298</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/plus.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Attention_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>324</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
      </property>
      <property name="text">
       <string>Введите соответствующие размеры по клику</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
      </property>
      <property name="text">
       <string>на отмеченных площадках</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>CenterVertSize</zorder>
  <zorder>CenterHorSize</zorder>
  <zorder>CenterVertSizeNum</zorder>
  <zorder>CenterHorSizeNum</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>Back_2</zorder>
  <zorder>Next_2</zorder>
  <zorder>TopHorSize</zorder>
  <zorder>RightVertSize</zorder>
  <zorder>LeftVertSize</zorder>
  <zorder>BottomHorSize</zorder>
  <zorder>TopHorSizeNum</zorder>
  <zorder>pushButton</zorder>
  <zorder>RightVertSizeNum</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>BottomHorSizeNum</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>Attention_2</zorder>
  <zorder>LeftVertSizeNum</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
